sql/gnomex_db_ddl.sql

```diff
@@ -2175,10 +2179,6 @@ CREATE TABLE gnomex.Sample (
   CONSTRAINT FK_Sample_Request FOREIGN KEY FK_Sample_Request (idRequest)
     REFERENCES gnomex.Request (idRequest)
     ON DELETE NO ACTION
-    ON UPDATE NO ACTION,
-  CONSTRAINT FK_Sample_AppUser FOREIGN KEY FK_Sample_AppUser (idAppUser)
-    REFERENCES gnomex.AppUser (idAppUser)
-    ON DELETE NO ACTION
```

Comment:
I'll have to look more closely when I get back from class but I think we may want this foreign key constraint.  Only difference is it should point to idLibPrepPerformedBy instead of idAppUser but it should reference idAppUser on the AppUser table.  This is most likely a typo performed by me.